doc/importing-data.md

```diff
@@ -0,0 +1,53 @@
+# Importing data from the old site
+
+## Prerequisites
+
+- The app is running and the database has been initialized
+- You have CSV database dumps from the old MySQL database in the format
+  `vote_export_TIMESTAMP.csv` and `places_export_TIMESTAMP.csv`
```

Comment:
If the names are significant, it should probably be `votes_export_TIMESTAMP.csv` for consistency, unless the singular `vote_` is how it gets output by MySQL.